### ISSUE TITLE
kotlin-gradle-plugin version required by package not recent enough for react-native 0.71.0

### ISSUE THREAD

**Issue by RenaudAubert** [th3rdwave/react-native-safe-area-context#342]
Hi!
I've updated my app to `react-native- 0.71.0` and when I run `npx react-native run-android` I get this error
```
* What went wrong:
The Android Gradle plugin supports only kotlin-android-extensions Gradle plugin version 1.6.20 and higher.
The following dependencies do not satisfy the required version:
project ':react-native-safe-area-context' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10
```

I tried creating a new app and installed react-navigation (which requires this package) and I also get this error.

I'm on windows 10 using node 16.14.2

**Comment by Romick2005** [comment]
Suggested fix is to replace
`ReactNativeWebView_kotlinVersion=1.6.10`
with
`ReactNativeWebView_kotlinVersion=1.7.0`
https://github.com/th3rdwave/react-native-safe-area-context/blob/8f93c44f14a5ea4fd398457848abb75809d66ecd/android/gradle.properties#L1

**Comment by RenaudAubert** [comment]
> Suggested fix is to replace `ReactNativeWebView_kotlinVersion=1.6.10` with `ReactNativeWebView_kotlinVersion=1.7.0`
> 
> https://github.com/th3rdwave/react-native-safe-area-context/blob/8f93c44f14a5ea4fd398457848abb75809d66ecd/android/gradle.properties#L1

That's what I figured. I didn't create a PR because changing `RNSAC_kotlinVersion` might have side effect and I know nothing about android development 😅

**Comment by kkafar** [comment]
Hi, I might be wrong, but looking at [this code](https://github.com/th3rdwave/react-native-safe-area-context/blob/8f93c44f14a5ea4fd398457848abb75809d66ecd/android/build.gradle#L99) you can try to add following line to your project `gradle.properites`

``
kotlinVersion=1.6.20
```

**Comment by janicduplessis** [comment]
Fixed in 4.5.0
